src/js/clickable-component.js

```diff
@@ -121,14 +125,14 @@ class ClickableComponent extends Component {
    * @return {Component} The child component (created by this process if a string was used)
    * @method addChild
    */
-  addChild(child, options={}) {
-    // TODO: Fix adding an actionable child to a ClickableComponent; currently
+  addChild(child, options = {}) {
+    // TO_DO: Fix adding an actionable child to a ClickableComponent; currently
```

Comment:
videojs-standard doesn't want you leaving `TODO` comments around (which makes sense because they almost never get done and only cause confusion down the line), but I didn't want to simply _delete_ them. So, I changed them to `TO_DO` which is kind of ugly, I know.